### ISSUE TITLE
Wrong regression test for 933160 rule?

### ISSUE THREAD

**Issue by theMiddleBlue** [coreruleset/coreruleset#3198]
maybe there's a good reason, but I can't understand why we're doing this:

```shell
$ grep 'Print_r' tests/regression/tests/REQUEST-933-APPLICATION-ATTACK-PHP/933160.yaml 
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
            data: x=Print_r%28%20%29
```

If I'm understanding it well, all these tests pass just because we send always the same payload. Am I correct?

**Comment by theseion** [comment]
Even if not, many tests in that file are garbage. It's often unclear what is actually being tested.

**Comment by dune73** [comment]
The URIs are different with most tests. No need to make it a POST though.

**Comment by theseion** [comment]
Closed via #3462
